master/static/migrations/20210922170815_update-model-registry-get-apis.up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE public.modelsv2 (
```

Comment:
I am curious, why is it creating new tables instead of altering the old ones?

Reply:
We're changing the primary keys + foreign keys for the tables, and I felt it was easier in that case to just create new ones, copy the data, and delete the old ones. @mackrorysd do you think I should just be altering the tables?

Reply:
It does sound easier, but also more risky / messy. But I'll review the specifics of both tables before we meet.